projects/control-service/projects/job-builder/build_image.sh

```diff
@@ -52,6 +52,7 @@ cat > /kaniko/.docker/config.json <<- EOM
       "password":"$registry_password",
       "auth": "$auth"
     }
+  $extra_auth
```

Comment:
Is this coming from env variable? 

I think it's good idea to document the env variable at the top of the script. 


It can be 
```
# descritpion
# detailed description 
EXTRA_AUTH=${EXTRA_AUTH:-""}
```
is something I often. This way reader of the script knows what is the expected input.

Reply:
me too; is the comment already resolved ?

Reply:
yeah